tracing-tracy/Cargo.toml

```diff
@@ -3,7 +3,7 @@ name = "tracing-tracy"
 version = "0.10.4"
 authors = ["Simonas Kazlauskas <tracing-tracy@kazlauskas.me>"]
 license = "MIT/Apache-2.0"
-edition = "2018"
+edition = "2021"
```

Comment:
Is there a reason why you think this change is something we should do?

Reply:
No, this is more of a "we should be up-to-date" axium. The changes are listed in this chapter: https://doc.rust-lang.org/edition-guide/rust-2021/index.html.

_Why not?_ is probably a better question IMO.